docs/networking/vlan/common-linode-vlan-use-cases/index.md

```diff
@@ -0,0 +1,75 @@
+
+A VLAN is a private network. This means that devices on a VLAN can communicate only with each other and with devices which are on the physical network, but not with devices that are not part of the VLAN. In other words, broadcast, and multicast, packets sent from one entity in a VLAN only reach other entities that are present in the same VLAN on the same network. This enables you to implement access and security policies to a particular group of users.
+
+### Benefits
+
+VLAN helps you manage the network by segmenting the network, confining the broadcast domains, reducing network traffic, and enforcing security policies.
+
+## Use Cases
+
+Some of the most popular use cases for VLAN are as follows.
+
+### Secure Data and Traffic
+
+If you have sensitive data on your network, you can use a VLAN to help keep it secure. By isolating of the devices in the network with privileged data, you decrease the possibility of confidential information breaches. For example, the devices in an organization can be segmented into VLANs such as management, sales, support, guests, and others. The devices in one VLAN cannot access the files in another VLAN.
+
+Likewise, the traffic in the VLAN is secure. All packets sent over the network in the VLAN are private and protected from access by anyone outside of the network.
+
+#### Example: Web Server and Secure Database
+
+![Web Server to VLAN Secure Database Configuration](vlan-web-server-db-config.png "Web Server to VLAN Secure Database Configuration")
+
+In the diagram above, Linode 1 is a web server and has access to the internet over the `eth0` interface. It also has access to Linode 2 and the VLAN over the `eth1` interface. Linode 2 on the VLAN is a secure database that houses sensitive data. Communication between Linode 1 and 2 is transmitted over `eth1` and is secure and private.
+
+#### Example: Kubernetes Cluster
+
+![Kubernetes Cluster with VLAN Configuration](vlan-kubernetes-cluster-config.png "Kubernetes Cluster VLAN Configuration")
+
+In the diagram above, Linode 1 and 2 are both in a Kubernetes cluster. Both nodes have services exposed to the internet over the `eth0` interface and communication between Pods is kept private and secure on the VLAN over the `eth1` interface.
+
+### Cost Effective Segmentation
+
+VLANs reduce the cost of the networking by efficiently using the existing resources and bandwidth. You can build multiple VLANS within an existing network instead of building multiple networks. Segmenting a network into smaller VLANs costs less than creating separate routed networks with routers. Similarly, you can run a secure VLAN in the cloud that is protected from the rest of the internet that without generating extra outbound network transfer costs since internal VLAN traffic is internal and private.
```

Comment:
"...from the rest of the internet without generating..." (axe the "that")

Reply:
done